sdk/azcore/version.go

```diff
@@ -1,15 +0,0 @@
-//go:build go1.16
```

Comment:
Keeping a version.go file for each back and sub-package is cleaner.  Makes it easy for our automation and for customers to quickly identify package versions.

Reply:
The module versions as an atomic unit, not sub-packages.  The reason the version info moved to `internal/shared` is because it's used across multiple packages.

Reply:
Also, it was a conscious decision to _not_ expose these values in the code.

Reply:
I'm interested as to why we decided not to export.  That is different than other languages.